gordian/files/changelog_file.py

```diff
@@ -0,0 +1,113 @@
+from . import MarkdownFile
+import re
+from datetime import datetime
+
+CHANGELOG_ENTRY_REGEX = '.*\[([0-9]*\.[0-9]*\.[0-9]*)\] - [0-9]{4}-[0-9]{1,2}-[0-9]+'
```

Comment:
Right now this is not overridable, and I think it needs to be to work with different changelog formats. How should this is passed in? Config parameter?

Reply:
add a complex regexp in the command line like this is kind of messy. We can leverage the configuration file maybe to add an attribute with this, but we can create an issue for that, maybe someone from the community can contribute with it.

Reply:
Agreed - I wouldn't want it to be a CLI argument. I was imagining a Config object that could be configured with the regex.